packages/plugins/other/visitor-plugin-common/src/utils.ts

```diff
@@ -47,8 +47,12 @@ export function block(array) {
   return array && array.length !== 0 ? '{\n' + array.join('\n') + '\n}' : '';
 }
 
-export function wrapWithSingleQuotes(str: string | NameNode): string {
-  return `'${str}'`;
+export function wrapWithSingleQuotes(value: string | number | NameNode): string {
+  if (typeof value === 'number' || (typeof value === 'string' && !isNaN(parseInt(value)))) {
```

Comment:
This can be dangerous because `parseInt('2a')` will return 2 as in `parseInt('2')`

Reply:
The final code can be;
```ts
if (parseFloat(value).toString() === value.toString()) {
   return value; // Number
}
```